src/components/Card/Card.tsx

```diff
@@ -30,16 +31,45 @@ export const Card = ({
 
     const clickableProps = !!onClick ? mergeProps(buttonProps, focusProps) : {};
 
+    React.useEffect(() => {
```

Comment:
This seems a bit over-engineered, I believe we can simplify by using the useButton hook on line 26.
Try this:
On line 26:
```
const { buttonProps } = useButton(
        {
            elementType: 'div',
            onPress: (event) => onClick && onClick(event),
            onPressChange: (isPressed) => {
                setKeyDownTrue(isPressed);
            },
        },
        ref,
    );

```

And remove all useEffect for keyUp and KeyDown.

Reply:
Yep this works. Updated.